### PR TITLE
519 default content

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/519
Replaces https://github.com/backdrop/backdrop/pull/1566

### CODE REVIEW

**Comment by jenlampton** [comment]
tests not passing...
edit: oh, that's a random fail.

File: core/modules/node/tests/node.test [hunk 1]
```diff
@@ -525,7 +538,7 @@ class NodeCreationTestCase extends BackdropWebTestCase {
     $this->backdropPost('node/add/page', $edit, t('Save'));
 
     // Check that the user was redirected to the home page.
-    $this->assertText(t('Welcome to Backdrop'), 'The user is redirected to the home page.');
+    $this->assertText(t('This is your first post, edit it or delete it'), 'The user is redirected to the home page.');
```
Comment (quicksketch) [inline core/modules/node/tests/node.test]: Oh boy. This seems less than ideal check for the homepage. In any case, missing a period at the end of this sentence, which will cause a new translated string.

File: core/modules/views/tests/views_groupby.test [hunk 2]
```diff
@@ -48,9 +50,6 @@ class ViewsQueryGroupByTest extends ViewsSqlTest {
     $this->assertEqual($types[$type2->type], 3);
   }
 
-  //public function testAggregateSum() {
-  //}
```
Comment (quicksketch) [inline core/modules/views/tests/views_groupby.test:52]: Ha! :+1:

**Review by quicksketch — CHANGES_REQUESTED** [review]

**Review by quicksketch — APPROVED** [review]

**Comment by quicksketch** [comment]
Closing/reopening for new sandbox.

**Comment by quicksketch** [comment]
Something screwy with the sandbox not tearing down. This looks great and I tested it locally. Merged into 1.x for 1.5.0!

**Pull request merged**